src/apps/lwaftr/binding_table.lua

```diff
@@ -143,10 +143,31 @@ local BindingTable = {}
 function BindingTable.new(psid_map, br_addresses, softwires)
    local ret = {
       psid_map = assert(psid_map),
-      br_addresses = assert(br_addresses),
-      softwires = assert(softwires)
+      softwires = assert(softwires),
+      br_addresses = cltable.new({key_type = ffi.typeof('uint8_t[16]')})
```

Comment:
@xray7224 Nope, that's what actually it's necessary to do, isn't? (to store the br-address as a 16-byte IPv6 address in a softwire entry).